region/wreckedship/main/Phantoon's Room.json

```
@@ -175,6 +175,35 @@
       "note": "No ammo count because Supers are farmable here.",
       "devNote": "FIXME add tech/energy/item requirements to survive flames."
     },
+    {
+      "link": [1, 1],
+      "name": "Phantoon Microwave",
+      "requires": [
+        "Charge",
+        "Plasma",
+        {"ammo": {"type": "Missile", "count": 1}},
+        "canXRayWaitForIFrames",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"and": [
+            "canDodgeWhileShooting",
+            {"enemyDamage": {"enemy": "Phantoon", "type": "flame", "hits": 1}},
+            {"enemyDamage": {"enemy": "Phantoon", "type": "contact", "hits": 1}}
+          ]},
+          {"and": [
+            {"enemyDamage": {"enemy": "Phantoon", "type": "flame", "hits": 2}},
+            {"enemyDamage": {"enemy": "Phantoon", "type": "contact", "hits": 2}}
+          ]}
```

```suggestion
          {"and": [
            "canDodgeWhileShooting",
            {"enemyDamage": {"enemy": "Phantoon", "type": "flame", "hits": 2}},
            {"enemyDamage": {"enemy": "Phantoon", "type": "contact", "hits": 2}}
          ]}
```
Mainly 0 tank phantoon is too much for hard.
It also expects a lot of knowledge about the fight so set up the microwave kill.  Clearing random flames, shooting the fire ring when it spawns, and manipulating phantoon to come back on screen in a favourable position or just in a known position.  And that brings you most of the way to the missile only kill.